homeassistant/components/matter/sensor.py

```diff
@@ -635,7 +635,6 @@ def _update_from_device(self) -> None:
         entity_description=MatterSensorEntityDescription(
             key="NitrogenDioxideSensor",
             native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
-            device_class=SensorDeviceClass.NITROGEN_DIOXIDE,
             state_class=SensorStateClass.MEASUREMENT,
         ),
```

Comment:
Please add translations for this. You can use a reference to the sensor entity platform name. Check tuya for some examples